storage/integration_test.go

```diff
@@ -204,10 +205,10 @@ func initIntegrationTest() func() error {
 		if err != nil {
 			log.Fatalf("NewStorageControlClient: %v", err)
 		}
-		if err := client.Bucket(bucketName).Create(ctx, testutil.ProjID(), nil); err != nil {
+		if err := client.Bucket(bucketName).Create(ctx, testutil.ProjID(), &BucketAttrs{SoftDeletePolicy: &SoftDeletePolicy{RetentionDuration: 0}}); err != nil {
```

Comment:
I think this should maybe be a separate change in our test harness? Was it impossible to delete the bucket if this wasn't included?

Reply:
Oops, this is leftover.  My org has a policy set that made this necessary for me to be able to run the tests.  I'll remove this before merge.